src/main/java/org/web3j/deploy/Predeploy.java

```diff
@@ -8,5 +8,8 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.METHOD)
 public @interface Predeploy {
-    String network();
+    String profile();
+    String[] credentialKeys() default {};
```

Comment:
I don't think it's a good idea to have this in the annotation, as you're already writing code to set this up anyway, and having it in the annotation just complicates it